index.js

```diff
@@ -35,12 +43,14 @@ async function main() {
   }
   await getOtherInteractions();
   console.log("Other interactions fetched ✅");
-  await getSwapTovariableStats();
-  console.log("Swap to variable stats fetched ✅");
+  // await getSwapTovariableStats(); // deployer 21 is used for every swap to variable so we already have all the txs from it
```

Comment:
So maybe delete this or you prefer to keep it and deleted in other PR for easy search ?

Reply:
Yeah, I keep it if one day we have to use it again. But the `getSwapTovariableStats` remains here in any case so I can remove it if you want